src/Adapters/TraceListener.Shared/ApplicationInsightsTraceListener.cs

```diff
@@ -47,6 +47,16 @@ public ApplicationInsightsTraceListener(string instrumentationKey)
             this.TelemetryClient.Context.GetInternalContext().SdkVersion = SdkVersionUtils.GetSdkVersion("sd:");
         }
 
+        /// <summary>
+        /// Initializes a new instance of the ApplicationInsightsTraceListener class, specifying a TelemetryClient to use
+        /// </summary>
+        /// <param name="telemetryClient">TelemetryClient to use for tracing</param>
+        public ApplicationInsightsTraceListener(TelemetryClient telemetryClient)
```

Comment:
Yes I believe that would work. I'm confused however, in the documentation  (https://docs.microsoft.com/en-us/azure/application-insights/app-insights-api-filtering-sampling#add-properties) it says that ITelemetryInitializer specifies the Initialize() method while ITelemetryProcessor provides the Process() method. Did you mean ITelemetryProcessor in the above example?

And just to be clear, are you proposing a new constructor that takes a TelemetryConfiguration? Would you guys be doing that work or would be fine with me adding it? Thanks!